src/components/button-group/examples/button-group-composite.tsx

```diff
@@ -51,22 +51,24 @@ export class ButtonCompositeExample {
     public render() {
         return [
             <limel-button-group {...this.props} onChange={this.handleChange} />,
-            this.renderForm(),
             <limel-example-event-printer
                 ref={(el) => (this.eventPrinter = el)}
             />,
+            this.renderForm(),
```

Comment:
In the composite example for `limel-button`, the event printer is still below the form. I don't know where is better to put it, now that the form is no longer collapsible? I would probably still put it below the controls… 🤔

Reply:
Since this new component doesn't have a margin bottom (it's usually the last, and a margin bottom makes the UI of the example quite weird). So when we put it on top of event printer, it becomes even move weird, because there won't be any space between them at all. I'll improve the `example-event-printer` and `example-value` in another PR to avoid such issues, and then will move them back to their previous place. However, I also think in composite examples, events aren't the main topic? Playing around with settings is the main purpose. Or?

Reply:
🙊 😅

Reply:
⚡